themes/theme-b2b/src/components/Table.styles.ts

```diff
@@ -14,6 +14,7 @@ export const Table: ThemeComponent<'Table'> = {
         variant: {
           compact: 'py-2',
           expanded: 'py-6',
+          borderedTable: 'border-x-[1px] border-t-[1px]',
```

Comment:
- I thought we wanted to show all borders?
- Shouldn't it be kebab-cased? Maybe we can also call it `grid` or something.

Reply:
I call it like in core to make it consistent :) We have there all borders? or what you mean= :)

Reply:
@sebald ?

Reply:
I thought we also show the surrounding borders. I think we should stick to at least kebab case for naming, because this would end up in CSS :)